### ISSUE TITLE
What is the meaning of leaf_value?

### ISSUE THREAD

**Issue by heroxrq** [Microsoft/LightGBM#523]
@guolinke 
I trained a 3-classes classifier (labeled 0/1/2), and print the tree_digraph. However, I found that the leaf_value has negative values as bellow. What is the meaning of leaf_value? 
leaf_value:-0.0340909098529
leaf_value:-0.0656165804478
leaf_value:0.00399515747429

**Comment by guolinke** [comment]
The leaf value are the raw scores, not the probability. So it can be negative. 
And the probabilities are computed by softmax in multi-class.
